examples/qip-noisy-device-simulator.ipynb

```diff
@@ -10,11 +10,14 @@
     "\n",
```

Comment:
<p>for some reason this Latex equation is not rendering? maybe need to put on many lines</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/qutip/qutip-notebooks/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
I cannot really see which cell you are pointing to. If you are talking about `pulse.print_info()`, I guess LaTeX only renders a `Qobj` if it is the last object in the cell. I'm using python print so I don't think they won't be rendered.